builtInFunctions/baseAccountFreezer.go

```diff
@@ -8,11 +8,12 @@ import (
 
 	"github.com/ElrondNetwork/elrond-go-core/core"
 	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	guardiansData "github.com/ElrondNetwork/elrond-go-core/data/guardians"
 	"github.com/ElrondNetwork/elrond-go-core/marshal"
 	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
 )
 
-var guardianKeyPrefix = []byte(core.ElrondProtectedKeyPrefix + GuardiansKeyIdentifier)
+var guardianKeyPrefix = []byte(core.ElrondProtectedKeyPrefix + core.GuardiansKeyIdentifier)
```

Comment:
this is the entire key, not only the key prefix right?
if yes please rename to guardianKey